app/models/gemini_client.py

```
@@ -76,27 +77,87 @@ def ask(self, prompt: str):
+            response_dict = None
+            parsed = getattr(response, "parsed", None)
+            if parsed is not None:
+                try:
+                    # pydanticモデルやdictの場合に対応
+                    if isinstance(parsed, dict):
+                        response_dict = parsed
+                    elif hasattr(parsed, "dict"):
+                        response_dict = parsed.dict()
+                    else:
+                        response_dict = parsed
+                except Exception:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)